### DISCUSSION TITLE
SMTP smuggling Security issue

### DISCUSSION THREAD

**Discussion by prabhushan** [postalserver/postal#2736] [Help with using Postal]
https://www.postfix.org/smtp-smuggling.html - is it relevant in postal?

**Comment by willpower232** [comment]
I think Postal is handling newlines consistently so should be fine

https://github.com/postalserver/postal/blob/2f62baa238fc1102706ee4acf079b7a876b05283/lib/postal/smtp_server/server.rb#L173-L184
